server/api/batch_inventory.py

```diff
@@ -78,6 +80,26 @@ def batch_key_to_name(
     )
 
 
+def validate_choice_name_and_get_choice_id(
+    contest: Contest, choice_name: str
+) -> Optional[str]:
+    choice_id = None
+    for choice in contest.choices:
+        if choice.name == choice_name:
+            choice_id = choice.id
+            break
+
+    if (
+        not choice_id
+        and choice_name
+        and choice_name.lower() != "overvote"
+        and choice_name.lower() != "undervote"
+        and choice_name.lower() != "write-in"
```

Comment:
Are these special values the same in Hart CVRs as they are in ES&S CVRs?

Reply:
overvotes and undervotes are not returned as choices in the parse_contest_xml helper I'm using and Write-In was capitlized differently which is why I changed these to check the lower() version

Reply:
Ah, so is this filtering out write-ins? Should we be including them?